templates/tfengine/components/cicd/triggers/main.tf

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Leave a TODO to merge the 3 submodules.

Reply:
Already done, but one level up:

https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/949/files/46e2b9cc6f9761574e7fb116fdad481cb71f1f25#diff-cf13a242ea112a5f59317e958fce2c903c8815aa5080db140795310fb12e6d87R201